cache/cache.go

```diff
@@ -136,6 +141,46 @@ func collectIncludePaths(homeDir, projectDir string, cacheLevel cache.Level) ([]
 	return includePths, nil
 }
 
+/*
+If the indicator is version controlled in git and has changes, we create a copy of it with its original content.
+*/
+func prepareUnmodifiedIndicator(indicator string) string {
+	indicatorDir := filepath.Dir(indicator)
+	indicatorFile := filepath.Base(indicator)
+
+	cmd := exec.Command("git", "ls-files", "--error-unmatch", indicatorFile)
+	cmd.Dir = indicatorDir
+	m := command.NewWithCmd(cmd)
+	code, err := m.RunAndReturnExitCode()
```

Comment:
i would return the command error message in case of a failure, so that we can debug issues later

Reply:
We can log errors for debugging reasons however in most cases errors are expected and not considered to be "unexpected" behaviour